lib/irq.c

```diff
@@ -427,10 +428,10 @@ vfu_irq_message(vfu_ctx_t *vfu_ctx, uint32_t subindex)
     }
 
     irq_info.subindex = subindex;
-    ret = tran_sock_msg(vfu_ctx->conn_fd, msg_id,
-                        VFIO_USER_VM_INTERRUPT,
-                        &irq_info, sizeof irq_info,
-                        NULL, NULL, 0);
+    ret = vfu_ctx->trans->send_msg(vfu_ctx, msg_id,
```

Comment:
I compared against master, and I'm so confused as to why github thinks there's a change here.

Reply:
Not sure either, I also diff'ed against master and don't see your changes. I'll merge and if it breaks things I'll revert the merge.